transforms/acceptance/index.js

```diff
@@ -24,6 +24,27 @@ module.exports = function transformer(file, api) {
     })
     .toSource();
 
+  // register and inject helpers
+  code = j(file.source)
```

Comment:
@daibhin I think this should be code instead of file.source ? 🤔

Reply:
Yep, fixed. Sorry about that